app/models/work.rb

```diff
@@ -48,6 +46,9 @@ def self.options_for_select_box
             presence: true,
             inclusion: { in: Types.all }
 
+  validates :versions,
+            presence: true
+
```

Comment:
@rschenk So I assume that the UI will create the initial work and its version? If we do this outside of the UI, we'd just do `work.versions.build` or some such?